recipes/base_install.rb

```diff
@@ -18,7 +18,12 @@
 case node['platform_family']
 when 'rhel', 'amazon'
   include_recipe 'yum'
-  include_recipe "yum-epel" if node['platform_version'].to_i < 7
+  if node['platform_family'] == 'amazon' and node['platform_version'].to_i == 2
+    execute "amazon-linux-extras install -y epel"
```

Comment:
do we need `epel` for alinux2?

Reply:
Yes. It's used to install a lot of the base packages.